testnet/src/node.rs

```diff
@@ -125,11 +237,14 @@ impl Node {
             Ok(res) => res,
             Err(err) => panic!("Error while opening chain state at path {}: {:?}", config.get_chainstate_path(), err)
         };
-        let mut event_dispatcher = EventDispatcher::new();
 
-        for observer in &config.events_observers {
+        let (dispatcher_channel, dispatch_recv) = channel();
```

Comment:
Shouldn't this be `sync_channel(1)` or something?

Reply:
(same for all the `channel()` usages)

Reply:
So that the dispatcher stalls the relayer ?

Reply:
As of 67a21e3, the event dispatcher no longer has it's own thread and channel.